ci-operator/config/openshift-knative/backstage-plugins/OWNERS

```diff
@@ -1,21 +1,23 @@
-# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
```

Comment:
@pierDipi any idea where that change comes from? (I just updated some serving config in hack).